pre_commit_hooks/yamlfmt

```diff
@@ -63,6 +63,12 @@ class Cli:
             action="store_true",
             help="whether to keep existing string quoting"
             )
+        parser.add_argument(
+            "-e",
+            "--explicit-start",
+            action="store_true",
+            help="whether to add an explicit document start"
+            )
```

Comment:
```suggestion
        parser.add_argument(
            "-e",
            "--explicit-start",
            action="store_true",
            default=True,
            help="whether to add an explicit document start"
            )
```
To keep the old behavior.

Reply:
@linuxdaemon isn't handled in `yaml.explicit_start = kwargs.get("explicit_start", True)`?

Reply:
no, the get only applies if the kwarg isn't passed to the function, not if the argument isn't set.